### PR TITLE
Update identity.yml

### PR DESCRIPTION
**Pull request by Pentagrade**
1Password Supports 2FA through an Authentication app providing a 6 digit pin
image

### CODE REVIEW

**Comment by Pentagrade** [comment]
Ah, just realized they have U2F too now

File: _data/identity.yml [hunk 1]
```diff
@@ -4,6 +4,7 @@ websites:
       img: 1password.png
       tfa: Yes
       software: Yes
+      otp: Yes
```
Comment (alex-nitrokey) [inline _data/identity.yml:7]: Please add a 'hardware: Yes' above this line.

**Comment by Pentagrade** [comment]
@alex-nitrokey I've remade this pull request in #144 and included it there

**Comment by alex-nitrokey** [comment]
A, yes, sure, my fault :)

**Pull request closed without merging**